app/controllers/evms_controller.rb

```diff
@@ -63,11 +68,11 @@ def index
     # export
     respond_to do |format|
       format.html
-      format.csv {
+      format.csv do
         send_data(@project_evm.to_csv,
-                  :type => 'text/csv; header=present',
-                  :filename => 'evm_' + @project.name + '_' +Time.now.to_date.to_s + '.csv')
-      }
+                  type: 'text/csv; header=present',
+                  filename: 'evm_' + @project.name + '_' + Time.now.to_date.to_s + '.csv')
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.